addons/web_editor/static/src/components/media_dialog/video_selector.js

```diff
@@ -256,9 +256,11 @@ export class VideoSelector extends Component {
             div.innerHTML = `
                 <div class="css_editable_mode_display"></div>
                 <div class="media_iframe_video_size" contenteditable="false"></div>
-                <iframe frameborder="0" contenteditable="false" allowfullscreen="allowfullscreen"></iframe>
+                <iframe frameborder="0" contenteditable="false" allowfullscreen="allowfullscreen" title="Video"></iframe>
```

Comment:
Question: why was this change needed ? Maybe add an explanation in the commit message too :slightly_smiling_face:

Reply:
I removed this. That's not the topic of this PR. You're right.

Reply:
you could have kept it, I was just saying to explain it briefly :wink: Like "this commit also adds a title attribute to the video iframe, for accessibility (?) ." or something like that :slightly_smiling_face:

Reply:
That would be better with the title of the Video to enhance accessibility. I'll keep this in mind and maybe create a task for it. "Video" for the title of a video is a little bit light.